src/net/sourceforge/kolmafia/swingui/OptionsFrame.java

```diff
@@ -1187,9 +1187,9 @@ public SVNPanel() {
       this.queue(
           this.newTextArea(
               """
-	      Configure the behavior of Mafia's built-in SVN client here.
+        Configure the behavior of Mafia's built-in SVN client here.
```

Comment:
These look to be spotless adjustments.  But since this file was modified recently IIRC how come there are spotless changes?  Is spotless not consistent across all environments?  Does this mean there is an IDE somewhere that is not set properly for tabs vs spaces but spotless isn't always catching it?  Observation, not blocker.

Reply:
These weren't spotless adjustments -- while editing the file I got some warnings about tabs / spaces in the same line, which I fixed.